server/api/apiSpecification.yaml

```
@@ -0,0 +1,568 @@
+openapi: 3.0.0
+info:
+  description: >
+    _This document describes the REST API of Ark Automate._
+    Ark Automate is a platform that allows office users and software developers to automate business or everyday processes by simply sketching the steps of their process. By using simple flowcharts or powerful BPMN in their process outlines, users can create small software solutions using RPA that finish their tasks much faster and more reliably.
+  version: 1.0.0
+  title: Ark API
```

```suggestion
  title: Ark-Automate API
```

I like Ark-Automate Api better